style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: rgb(241, 48, 0);
+}
```

Comment:
- [ ] From the project requirement, your CSS file should include one class with styles for your header not a tag name with styles for your header. So please make sure that you use a CSS class selector to style your header. :pray: